example/pythonlib/web/1-hello-flask/foo/test/src/test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from foo import app  # type: ignore
+
+
+class TestScript(unittest.TestCase):
+    def setUp(self):
```

Comment:
Let's try to follow [python's styleguide](https://peps.python.org/pep-0008/#method-names-and-instance-variables) in our examples.

This should be snake_case "`set_up`"

Reply:
ya i will update for every example in a separate pr(will raise shortly)

Reply:
Actually i am concerned about django way of testing and migrations am i doing this right way ?

Reply:
@jodersky i think you are not aware of this thing actually we are overriding the setUp and tearDown from the Upper class `Unitest.Testcase`

upon going deeper i got the actual code for `Testcase` Library and there i found that these are the overridden functions

https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp 

so we can't change names for this case

Hope you Understand!

Reply:
ah yes, that makes sense